ansible/roles/nova/templates/nova.conf.j2

```diff
@@ -103,7 +103,7 @@ enabled = false
 {% if enable_nova_serialconsole_proxy | bool %}
 [serial_console]
 enabled = true
-base_url = {{ nova_serialproxy_protocol }}://{{ nova_serialproxy_fqdn }}:{{ nova_serialproxy_port }}/
+base_url = {{ nova_serialproxy_protocol }}://{{ kolla_external_fqdn }}:{{ nova_serialproxy_port }}/
```

Comment:
I don't think this is necessary since this seems to be the default value anyway:

https://github.com/openstack/kolla-ansible/blob/stable/stein/ansible/group_vars/all.yml#L332